### DISCUSSION TITLE
does isar support  x86 emulators?

### DISCUSSION THREAD

**Discussion by coodin** [isar/isar#938] [Help]
I have seen this problem in another issue and it have been closed but according to my experience the problem still remain. and the problem  was  32-bit devices: Unhandled Exception: IsarError: Cannot open Environment,  here is the link your can reach https://github.com/isar/isar/issues/187, according to Simon Leier this issue should have been solved Dart2.17 and next Flutter relase 
(his asnswer is here https://github.com/isar/isar/issues/187#issuecomment-1107813273), so my question is  does isar support x86 emulators? thank you

**Comment by simc** [comment]
It doesn't and I don't want to spend time fixing x86 emulator support since I see no use case for them. Happy to review contributions though.
